src/ocean/util/app/DaemonApp.d

```diff
@@ -292,6 +292,10 @@ public abstract class DaemonApp : Application,
             Signal to trigger reopening of files which are registered with the
             ReopenableFilesExt. (Typically used for log rotation.)
 
+            If the command to reopen log files is sent via a UNIX socket, then
```

Comment:
Suggestion: "If this field is set to 0, no signal handler will be installed for file reopening. This is useful if your app is deployed to use a different means of triggering file reopening (e.g. a UNIX socket command)."

Reply:
Sounds good.